.env-dev

```diff
@@ -0,0 +1,11 @@
+DJANGO_DEBUG=True
```

Comment:
In our other repos, we've simply called this file `.env`, which defines environment variables for local development. Would you mind following that standard? Unless there's a good reason not to.

Reply:
It can be renamed.